### PR TITLE
New CL Suit

### PR DESCRIPTION
**Pull request by Hughgent**
:cl: Hughgent & Manezinho
add: New CL suit, Admin spawn only. You gotta earn it.
/:cl:

[why]: I weep at how beautiful it is, and yet I will never get to wear it.

liason galaxy

### CODE REVIEW

**Comment by Shadow-Quill** [comment]
```
Attention TGS Theseus:

We have been prompted by a recent fax from your Corporate Liaison to look back through
our records of previous faxes sent. There have been numerous faxes sent to us by this Liaison,
all of which are extremely well written. Even I was nearly brought to tears by the sheer
beauty of all these faxes.

In light of this accomplishment, the Corporate Liaison has been granted ownership to an extremely
exclusive suit. Designed by Nanotrasen's Finest, we guarantee the suit is a soft as a civilian
shuttle in transit from a planet to it's moon. It is one of very few in existence, and should be
considered in high regard. Should anyone attempt to forcefully remove it from the Liaison, we
shall be Not Happy.

Congratulations.

Johnathan Ward
Head of Nanotrasen Communications
```

I love how it looks!

**Comment by Alterist** [comment]
:shrug:

**Comment by Hughgent** [comment]
Manezinho is working on a reddish version as well.

**Comment by Hughgent** [comment]
reddish version is here.
liason galaxy

**Comment by Hughgent** [comment]
working as intended. If an Admin wants to reward the CL, they need only to set the briefcases variable max_points to 51. (and set points to 51 as well if they are feeling kind.)

image

**Comment by Borisvanmemes** [comment]
Theres a large issue with it being in the briefcase, the briefcase is seemly admin only and does not in fact spawn roundstart on the CL or in the CL office.

**Comment by Hughgent** [comment]
What do you mean it doesn't spawn?

image

**Comment by Borisvanmemes** [comment]
Not the same briefcase, thats one with a shity lock on it.

**Comment by Hughgent** [comment]
Latest master.
image
walking over to that briefcase.
image
picking it up
image
activate held object
image
it is that briefcase. I declare you an enemy of Nanotrasen!

**Comment by Borisvanmemes** [comment]
bloody fuck last round i tried it and it was just a shity locked one.

**Comment by Hughgent** [comment]
musta been the Admemes knowing you were a syndicate spy.

**Comment by Borisvanmemes** [comment]
Unless it just comes with the lock and that's just a part of it, guess the CL was just...not very smart.

File: code/game/objects/items/devices/portable_vendor.dm [hunk 1]
```diff
@@ -239,5 +239,7 @@
...
 							list("Ultrazine Pills", 20, /obj/item/storage/pill_bottle/restricted/ultrazine, "white", "Highly-addictive stimulant. Enhances short-term physical performance, particularly running speed. Effects last approximately 10 minutes per pill. More than two pills at a time will result in overdose. Withdrawal causes extreme discomfort and hallucinations. Long-term use results in halluciations and organ failure. Conditional distribution secures subject compliance. Not for personal use."),
 							list("Cash", 5, /obj/item/spacecash/c1000, "white", "$1000 USD, unmarked bills"),
-							list("Cigars", 5, /obj/item/storage/fancy/cigar, "white", "Case of premium cigars, untampered.")
+							list("Cigars", 5, /obj/item/storage/fancy/cigar, "white", "Case of premium cigars, untampered."),
+							list("De Void of Soul", 51, /obj/item/clothing/under/liaison_suit/galaxy_blue, "white", "The latest in ultrafashion. for those with a cool demeanor."),
```
Comment (DominikPanic) [inline code/game/objects/items/devices/portable_vendor.dm]: Not sure if this should be something available through regular means due to how disruptive it is visually. Seems better fit for an admin-only CL reward.

Comment (Hughgent) [reply code/game/objects/items/devices/portable_vendor.dm]: max is normally 50, these cost 51. The briefcase needs to be var-edited to have max_points = 51 for them to even be available. See the screen shot in a comment I made four days ago.

Comment (DominikPanic) [reply code/game/objects/items/devices/portable_vendor.dm]: Me stupid thanks Hugh

**Pull request merged**